packages/editor/src/client/atoms/descriptionList.tsx

```diff
@@ -20,9 +21,11 @@ export function DescriptionList({children}: DescriptionListItemProps) {
 export interface DescriptionListItemProps {
   label?: ReactNode
   children?: ReactNode
+  message?: ReactNode
+  type?: ReactNode
```

Comment:
Can you name this `messageType`? Just so it's clear what's being given a type here.